ROADMAP.md

```diff
@@ -0,0 +1,52 @@
+# ROADMAP
```

Comment:
I suggest put ROADMAP in issue or discussion, making it easier for code maintainance. 
Going through this PR, of course, the contribution is wonderful, but technically, I would say, less is more.

Reply:
Done!